app/dashboard/views.py

```diff
@@ -459,6 +459,52 @@ def post_comment(request):
             'msg': 'Finished.'
         })
 
+
+@require_POST
+def post_comment_project(request):
+    profile_id = request.user.profile if request.user.is_authenticated and hasattr(request.user, 'profile') else None
+    if profile_id is None:
+        return JsonResponse({
+            'success': False,
+            'msg': '',
```

Comment:
Let's add relevant message! kinda like https://github.com/gitcoinco/web/blob/master/app/dashboard/views.py#L5196-L5235

Reply:
hmm, this was missed as apart of things i removed, its not ready yet